python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -644,6 +644,20 @@ keys:
             delay_mechanism:
               type: str
               valid_values: ["e2e", "p2p"]
+            profile:
+              type: dict
+              keys:
+                g8275_1:
+                  type: dict
+                  keys:
+                    destination:
```

Comment:
I think we should collapse into `destination_mac_address`.

Reply:
We could even collapse further, since there are no other options under profile, but maybe it is good to keep those more open.

Reply:
yeah, I've flattened it to `destination_mac_address`, but I think its better to leave the option of additional profiles and options open